pages/trust/_legal-docs.html

```diff
@@ -0,0 +1,25 @@
+<div class="row row-page-header">
+  <div class="col-xs-12">
+    <ul style="margin-left: -20px;">
+      <li>
+        <a href="/terms">Terms of Service</a>
+      </li>
+      <li>
+        <a href="/legal/cookie-policy">Cookie Policy</a>
+      </li>
+      <li>
+        <a href="/legal/privacy-policy">Privacy Policy</a>
+      </li>
+      <li><a href="/legal/dpa">Data Processing Agreement</a> (see below)</li>
```

Comment:
I think the `(see below)` was in the document just for us to "see below for the details of the Data Processing Agreement page" - I don't think it needs to appear here. If it does need to be here, then I'm not sure what we're referring to.

Reply:
@tsmit291 ☝️

Reply:
yeah we don't need the (see below) text.